docs/source/api/apollo-gateway.mdx

```diff
@@ -3,24 +3,27 @@ title: "API Reference: @apollo/gateway"
 description: Apollo Gateway API reference
 ---
 
-This API reference documents the exports from the `@apollo/gateway`.
-
-
 ## `ApolloGateway`
 
-The core of the federated gateway implementation. For an example, see the ["implementing"](/federation/implementing/) section.
+The core class of Apollo Server's federated gateway implementation. For an
+example of using `ApolloGateway`, see [Implementing a federated graph](/federation/implementing/).
 
 ### `constructor(options)`: `ApolloGateway`
 
 #### Parameters
 
 * `options`: `Object`
 
-  * `serviceList`: `ServiceDefinition[]` _(must **not** be set when using [managed federation](https://www.apollographql.com/docs/platform/federation/#managed-configuration); **required otherwise**)_
+  An object that contains configuration options for the gateway.
+
+  Supported options include:
+
+  * `serviceList`: `ServiceDefinition[]`
 
-    > To reduce the possibility of different server instances using different configurations during a roll-out, we recommend using a managed configuration that provides a single single source of truth for a graph.  When using [Graph Manager](https://www.apollographql.com/docs/platform/graph-manager-overview/)'s [managed federation](https://www.apollographql.com/docs/platform/federation/#managed-configuration) feature, this is handled automatically and `serviceList` should not be specified.
+    > **Do not provide this option if you are using [managed federation](https://www.apollographql.com/docs/platform/federation/#managed-configuration).** If you
```

Comment:
We should be precise about terminology — In AGM docs, we often say that "managed federation" is just registering your services & running checks, and "managed configuration" is the idea that AGM manages your gateway's configuration. I'm not particularly tied to any naming scheme, but I do think that consistency is important :) we should either remove uses of "managed configuration" or use that term imho